app/lib/features/pins/pages/pins_page.dart

```diff
@@ -78,7 +80,7 @@ class PinsPage extends ConsumerWidget {
           itemCount: pins.length,
           gridDelegate: SliverGridDelegateWithFixedCrossAxisCount(
             crossAxisCount: max(1, min(widthCount, minCount)),
-            mainAxisExtent: 100,
+            mainAxisExtent: 170,
```

Comment:
@gtalha07 
Can you share the screen recording of the desktop app with the resizing screen flow? I have still doubts about fixed values that are updated.